week_3/15_apr/iterators.js

```diff
@@ -0,0 +1,15 @@
+console.log([1,-3,4].filter((value, index) => {
+    if (value >= 0) {
+        return true
+    }
+
+    else {
+        return false
+    }
+}));
+
+console.log([1,-3,4].map((value, index) => {
+    return value + 10
```

Comment:
@codal-kcolaco What is the btw foreach and map function? and when to use what?

Reply:
**forEach** is used to iterate over each element in the array, It is typically used when you want to perform some operation for each element of an array without necessarily transforming the array itself.

**map** creates a new array populated with the results of a provided function on every element in the array. Eg: Add 10 to each element of the array.